net/peer.go

```diff
@@ -137,6 +137,21 @@ func (p *Peer) Start() error {
 	p.mu.Lock()
 	defer p.mu.Unlock()
 
+	// reconnect to known peers
+	for _, id := range p.host.Peerstore().PeersWithAddrs() {
+		go func(id peer.ID) {
+			addr := p.host.Peerstore().PeerInfo(id)
+			err := p.host.Connect(p.ctx, addr)
+			if err != nil {
+				log.Info(
+					p.ctx,
+					"Failed to reconnect to peer`",
```

Comment:
todo: Why is there a backtick after peer?

suggestion: Change to this perhaps:
```suggestion
					"Failure while reconnecting to a known peer",
```

Reply:
✅